src/util.ts

```diff
@@ -428,3 +428,14 @@ export function getImportPath(text: string): string {
 
 	return '';
 }
+
+export function readDir(path): Promise<string[]> {
+	return new Promise<string[]>((resolve, reject) => {
+		fs.readdir(path, (err, files) => {
```

Comment:
instead of getting all files under the path and then checking for `main.go`, isn't it simpler to directly check for `path + 'main.go'` using `fs.statSync(filePath).isFile()` ?